src/js/components/SchemaForm.js

```diff
@@ -136,10 +136,10 @@ class SchemaForm extends mixin(StoreMixin) {
     }
 
     return (
-      <div className="form-row-element">
+      <div className="form-row-element"
+        prop={prop}>
```

Comment:
This is needed to find the exact place to add a new row. If this is not present the fields get added at the beginning of the form rather then above the button. This is only needed in the case if there isn't any row left.